src/Functions/toTime.cpp

```diff
@@ -10,7 +10,42 @@ using FunctionToTime = FunctionDateOrDateTimeToSomething<DataTypeDateTime, ToTim
 
 REGISTER_FUNCTION(ToTime)
 {
-    factory.registerFunction<FunctionToTime>();
+    FunctionDocumentation::Description description_toTime = R"(
+Converts a date with time to a certain fixed date, while preserving the time.
+
+If the `date` input argument contained sub-second components, they will be dropped in the returned `DateTime` value with second-accuracy.
+    )";
+    FunctionDocumentation::Syntax syntax_toTime = R"(
+toTime(date[, timezone])
+    )";
+    FunctionDocumentation::Arguments arguments_toTime = {
+        {"date", "Date to convert to a time. [`Date`](../data-types/date.md)/[`DateTime`](../data-types/datetime.md)/[`DateTime64`](../data-types/datetime64.md)."},
+        {"timezone", "Optional. Timezone for the returned value. [`String`](../data-types/string.md)."}
+    };
+    FunctionDocumentation::ReturnedValue returned_value_toTime = "Returns DateTime with date equated to `1970-01-02` while preserving the time. [`DateTime`](../data-types/datetime.md).";
+    FunctionDocumentation::Examples examples_toTime = {
+        {"Convert DateTime64 to time", R"(
+SELECT toTime(toDateTime64('1970-12-10 01:20:30.3000',3)) AS result, toTypeName(result)
```

Comment:
Let's use a date from this century as an example date.

Also, I don't think we need to overcomplicate the example, a DateTime instead of DateTime64 would do the job.